src/docs/implementations/dynatrace.adoc

```diff
@@ -11,6 +11,52 @@ include::install.adoc[]
 
 == Configuring
 
+=== Quick setup
+
+For setting up new integrations with Dynatrace, it is recommended to use the latest version of the API (`api-version: v2`).
+
+Use the following code in your project to export Micrometer metrics to the specified Dynatrace endpoint. 
+https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[The Dynatrace API token documentation] contains more information on how to create an API token.
+The metrics ingest (`metrics.ingest`) permission is required on the token in order to ingest metrics.
```

Comment:
```suggestion
The 'Ingest metrics' (`metrics.ingest`) permission is required on the token in order to ingest metrics.
It is recommended to limit scope to only this permission.
```

Reply:
Can we link to the API documentation when mentioning MINT first?